---
title: "Dual randomization of oligonucleotides to reduce the bias in ribosome-profiling libraries"
author: "Aarón Lecanda, Benedikt S. Nilges, Puneet Sharma, Danny D. Nedialkova, Juliane Schwarz, Juan M. Vaquerizas, Sebastian A. Leidel"
date: 2016-07-19
excerpt: ""
draft: false
tags:
  - Ribosome profiling
  - Cycloheximide
layout: single
links:
- icon: file-pdf
  icon_pack: fa
  name: PDF
  url: publications_pdf/lecanda_a_et_al_methods_2016.pdf
- icon: doi
  icon_pack: ai
  name: Journal
  url: https://www.sciencedirect.com/science/article/pii/S1046202316302201
---

# Abstract

Protein translation is at the heart of cellular metabolism and its in-depth characterization is key for many lines of research. Recently, ribosome profiling became the state-of-the-art method to quantitatively characterize translation dynamics at a transcriptome-wide level. However, the strategy of library generation affects its outcomes. Here, we present a modified ribosome-profiling protocol starting from yeast, human cells and vertebrate brain tissue. We use a DNA linker carrying four randomized positions at its 5′ end and a reverse-transcription (RT) primer with three randomized positions to reduce artifacts during library preparation. The use of seven randomized nucleotides allows to efficiently detect library-generation artifacts. We find that the effect of polymerase chain reaction (PCR) artifacts is relatively small for global analyses when sufficient input material is used. However, when input material is limiting, our strategy improves the sensitivity of gene-specific analyses. Furthermore, randomized nucleotides alleviate the skewed frequency of specific sequences at the 3′ end of ribosome-protected fragments (RPFs) likely resulting from ligase specificity. Finally, strategies that rely on dual ligation show a high degree of gene-coverage variation. Taken together, our approach helps to remedy two of the main problems associated with ribosome-profiling data. This will facilitate the analysis of translational dynamics and increase our understanding of the influence of RNA modifications on translation.